### ISSUE TITLE
Avoid git noise generated by modified files on Play

### ISSUE THREAD

**Issue by dalkia** [decentraland/unity-explorer#517]
### **Issue Description:**
The following files are modified every time the play button is pressed.

GenesisSkybox.mat
M_ChargeBarV2.mat
ProjectSettings.asset
LogoVideoTexture.renderTexture
EMPTY_DEFAULT_WEARABLE.prefab

We should fix it to avoid git noise

**Comment by m3taphysics** [comment]
Hey @mihakrajnc I believe you fixed some of these already?

**Comment by mihakrajnc** [comment]
Yes ChargeBar and BackgroundMaterial (not listed here), but I don't get any of the other changes so I assume those got fixed along the way. PR needs another dev approval: #3169
